test/sqlite/test-sqlite.mjs

```diff
@@ -0,0 +1,85 @@
+import '../common/index.mjs';
+
+import assert from 'node:assert';
+import path from 'node:path';
+import sqlite from 'node:sqlite';
+
+function resolveBuiltBinary(binary) {
+  return path.join(path.dirname(process.execPath), binary);
+}
+
+const binary = resolveBuiltBinary('libsqlite_extension');
+
+{
```

Comment:
Please don't use block scopes for tests. Either put these quasi-tests in separate files or use `node:test`. If you decide to use separate files, please also leave comments explaining what is being tested. If you use `node:test`, the test description should be enough.